packages/wrangler/src/publish.ts

```diff
@@ -20,8 +20,8 @@ type Props = {
   public?: string;
   site?: string;
   triggers?: (string | number)[];
-  zone?: string;
   routes?: (string | number)[];
+  legacyEnv?: boolean;
```

Comment:
Should we be incorporating legacy-_anything_ at this point? I know I'm not involved here but I assumed wrangler@2 was a bunch of new stuff but also a way to introduce some breaking changes thanks to the new major

Reply:
It's really easy to support this, since all of our APIs are the same for `/scripts/:name` and `/services/:name/environments/:name` -- that's why we designed it that way. We're still figuring out a migration system from old environments to service environments, though it will likely require some API work.

Reply:
That being said, I think service environments should be the default when 2.0 lands.

Reply:
We are not going to be breaking wrangler environments in wrangler 2.0. There will be plenty of people who won't migrate to the new services immediately, and wrangler 2 will be their path to migrating it. So we need to support it for a while.